specification/dartLangSpec.tex

```diff
@@ -16334,6 +16344,17 @@ \subsection{Identifier Reference}
   then $e$ evaluates to the function object obtained by closurization
   (\ref{functionClosurization})
   of $D$.
+\item
+  If $D$ is an instance getter declaration in an extension $E$,
+  then $e$ evaluates to the result of invoking said getter
+  with the current binding of \THIS{}, and
+  the current bindings of the type parameters declared by $D$.
+\item
+  If $D$ is an instance method declaration in an extension $E$
+  with type parameters \List{X}{1}{s},
+  then $e$ evaluates to the result of the extension method closurization
+  \code{$E$<\List{X}{1}{s}>(\THIS).\id}
+  (\ref{extensionMethodClosurization}).
 \item
   If $D$ is a local variable $v$
   (\commentary{which can be a formal parameter})
```

Comment:
Same comments about mentioning `enum` above and 
> ```latex
> \item
>   \commentary{%
>     A lexical lookup will never yield a declaration $D$
>     which is an instance member.%
>   }
> ```
looking wrong.

Reply:
I don't understand why I'd need to mention enumerated types in the item about local variables?

The reason why the lexical lookup will always yield a member signature rather than a declaration in the case where the selected entity is an instance member is unchanged from line 16260.

Reply:
Referring to line 16216:
> ```latex
>  If $D$ declares a class, mixin, type alias or type parameter,
>  the static type of $e$ is \code{Type}.
> ```
which should also mention `enum` declarations.